src/components/Text/Text.tsx

```diff
@@ -94,11 +97,12 @@ const Text = ({
         <Icon aria-hidden="true" path={mdiLoading} size={size} spin={1} />
       </StyledIconContainer>
     )}
-    <Skeleton color={color} StyledContainer={SkeletonContainer} isLoading={isLoading}>
+    <Skeleton isLoading={isLoading} skeletonRef={skeletonRef} StyledContainer={StyledSkeletonContainer} {...SkeletonProps}>
```

Comment:
looks like we lost the color prop here. It should still be set here by default, but now it will be able to be overwritten by passing `color` into the skeletonProps prop